nncf/quantization/algorithms/fast_bias_correction/algorithm.py

```diff
@@ -199,10 +206,22 @@ def reshape_bias_shift(self, bias_shift: TTensor, bias_value: TTensor, channel_a
         if bias_value.ndim > 1:
             new_shape = [1] * bias_value.ndim
             new_shape[channel_axis] = bias_shift.shape[0]
-            bias_shift = self._backend_entity.reshape_tensor(bias_shift, new_shape)
+            bias_shift = bias_shift.reshape(*new_shape)
         return bias_shift
 
-    def _get_fp_inputs(self, statistic_points: StatisticPointsContainer, node_name: str) -> Tuple[List, List]:
+    @staticmethod
+    def _get_bias_shift_magnitude(current_bias_value: NNCFTensor, updated_bias_value: NNCFTensor) -> float:
+        backend = current_bias_value.backend
```

Comment:
A reference to the backend interface (of abstract, well-defined type `NNCFTensorBackend`) is accessible from each NNCFTensor. Accessing this backend adds an extra line of code indeed, but it is not much more different than an extra line of `import numpy as np` that one had to do before.

Furthermore, if one does not want to do explicit calls to the backend when trying to call an operation on an `NNCFTensor`, he or she can almost always implement an equivalent method on the `NNCFTensor` itself and use that instead, e.g. if the following syntax seems to verbose:
```python
assert isinstance(nncf_tensor, NNCFTensor)
backend = nncf_tensor.backend
m = backend.mean(nncf_tensor, axis=(0, 2))
```
one could do:
```
assert isinstance(nncf_tensor, NNCFTensor)
m = nncf_tensor.mean(axis=(0, 2))
```